data_analysis/profit_analysis.py

```diff
@@ -8,10 +8,8 @@ def sell_today(value_today, value_tomorrow, no_units, daily_cost):
     """Returns a boolean value for to indicate whether to sell today."""
     profit_today = value_today * no_units
     profit_tomorrow = (value_tomorrow * no_units) - daily_cost
-
     return profit_today > profit_tomorrow
 
-
```

Comment:
Can you add this line back in please, there should be two blank lines between each function, I think the black autoformatter should have caught this, so you might not be using pre-commit hooks?